src/containers/BooksForm.js

```diff
@@ -8,21 +13,48 @@ const BooksForm = () => {
     'Learning',
     'Sci-Fi',
   ];
+  const [state, setState] = useState({ title: '', category: '' });
+  let error = false;
+
+  const handleChange = ({ target: { name, value } }) => {
+    setState({ ...state, [name]: value });
+  };
+  const { title, category } = state;
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (title === '' || category === '') {
+      error = true;
+    } else {
+      createBook({
+        id: Math.round(Math.random() * 100) + 1,
+        title,
+        category,
+      });
+      setState({ title: '', category: '' });
```

Comment:
- [ ] [REQUIRED] The `category value` should be equal to `Action`

Reply:
Done